css/style.css

```diff
@@ -0,0 +1,456 @@
+  }
+}
+
+@media (min-width: 769px) and (max-width: 986px) {
+
+  .menu { margin: 0 0 0 70px; }
+
+  .menu__select-level { margin-bottom: 70px; }
+
+  .level_btn {
+    margin: 0 0 10px 0;
+    font-size: 30px;
+    line-height: 40px;
+  }
+
+  .level_active::before {
+    top: 15px;
+    width: 15px;
+    height: 15px;
+  }
+
+  .start-button {
+    font-size: 30px;
+    line-height: 40px;
+    height: 70px;
+    width: 250px;
+  }
+
+  .pictures { display: none; }
+ 
+  .card-wrap,
+  .card__back,
+  .card__game-over,
+  .card__winner {
+    background-size: 175px 270px;
+    height: 270px;
+    width: 175px;
+  }
+
+  .three-card {
+    height: 270px;
+    width: 575px;
+  }
+
+  .six-card {
+    height: 580px;
+    width: 575px;
+  }
+
+  .ten-card .card-wrap,
+  .ten-card .card__back,
+  .ten-card .card__game-over,
+  .ten-card .card__winner {
+    background-size: 120px 170px;
+    height: 170px;
+    width: 120px;
+  }
+
+  .ten-card {
+    height: 360px;
+    width: 640px;
+  }
+}
+
+@media (min-width: 481px) and (max-width: 768px) {
+
+  .menu { margin: 0 0 0 50px; }
+
+  .title {
+    font-size: 40px;
+    line-height: 50px;
+  }
+
+  .menu__select-level { margin-bottom: 50px; }
+
+  .level_btn {
+    margin: 0 0 10px 0;
+    font-size: 30px;
+    line-height: 40px;
+  }
+
+  .level_active::before {
+    top: 15px;
+    width: 15px;
+    height: 15px;
+  }
+
+  .start-button {
+    font-size: 30px;
+    line-height: 40px;
+    height: 70px;
+    width: 250px;
+  }
+
+  .pictures { display: none; }
+
+  .card-wrap,
+  .card__back,
+  .card__game-over,
+  .card__winner {
+    background-size: 145px 230px;
+    height: 230px;
+    width: 145px;
+  }
+
+  .three-card {
+    height: 230px;
+    width: 455px;
+  }
+
+  .six-card {
+    height: 470px;
+    width: 455px;
+  }
+
+  .ten-card .card-wrap,
+  .ten-card .card__back,
+  .ten-card .card__game-over,
+  .ten-card .card__winner {
+    background-size: 85px 140px;
+    height: 140px;
+    width: 85px;
+  }
+
+  .ten-card {
+    height: 290px;
+    width: 465px;
+  }
+}
+
+@media (min-width: 320px) and (max-width: 480px) {
+
+  .menu { margin: 0 0 0 30px; }
+
+  .title {
+    font-size: 35px;
+    line-height: 45px;
+    margin: 30px 0 15px 0;
+  }
+
+  .subtitle { margin-bottom: 30px; }
+
+  .menu__select-level { margin-bottom: 40px; }
+
+  .level_btn {
+    margin: 0 0 5px 0;
+    font-size: 25px;
+    line-height: 35px;
+  }
+
+  .level_active::before {
+    top: 12px;
+    width: 12px;
+    height: 12px;
+  }
+
+  .start-button {
+    font-size: 25px;
+    line-height: 35px;
+    height: 60px;
+    width: 230px;
+  }
+
+  .pictures { display: none; }
+
+  .card-wrap,
+  .card__back,
+  .card__game-over,
+  .card__winner {
+    background-size: 85px 135px;
+    height: 135px;
+    width: 85px;
+  }
+
+  .three-card {
+    height: 135px;
+    width: 275px;
+  }
+
+  .six-card {
+    height: 290px;
+    width: 275px;
+  }
+
+  .ten-card {
+    height: 555px;
+    width: 275px;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5841733)